src/Trace/Sampler/Sampler.php

```diff
@@ -185,13 +185,15 @@ protected function parsedAndUpdateSettings(mixed $settings): ?Settings
                 $this->ready->map(static fn (): bool => true);
             }
 
-            //            if (!empty($parsed['warning'])) {
-            //                // $this->logger->warn($parsed['warning']);
-            //            }
+            if (!empty($parsed['warning'])) {
+                $this->logWarning($parsed['warning']);
+            }
+
             return $parsed['settings'];
         }
 
-        //            $this->logger->debug('Invalid settings', $settings);
+        $this->logDebug('Invalid settings', $settings);
```

Comment:
Should this be with a `.` like in OboeSampler, or is comma ok too?

```suggestion
        $this->logDebug('Invalid settings' . $settings);
```

Reply:
Yes, it is a typo. Thanks for catching that!